docs/usage/advanced_configuration.rst

```diff
@@ -0,0 +1,27 @@
+You can override default values of service deployment via ansible group_vars:
```

Comment:
Adding this file likely breaks the doc build because Sphinx detects an RST file which isn't part of any index, no?

Reply:
May be, this PR is just a draft to discuss open discussion for external_values. Related to #253.

Reply:
Now this PR is no more a draft. Need to add an entry in the index

Reply:
the PR come with a proposal